File: speed-loop.js
function main(){
  const cycles = 1000000000 // 1,000,000,000
  let start = Date.now()
  
  for(let i = 0; i < cycles; i++){
    // Empty loop
  }
  
  let end = Date.now()
  let duration = (end - start) / 1000

  console.info(`Javascript looped ${cycles} in ${duration} seconds`)
}

main();

File: speed-loop.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	for i := 1; i < 1000000000; i++ {
		// Empty lopp
	}
	end := time.Now()

	elapsed := end.Sub(start)

	fmt.Println(elapsed)
}
